### ISSUE TITLE
Add optional initial view for service definition

### ISSUE THREAD

**Issue by zstadler** [nextgis/quickmapservices_server#41]
For some services with minZoom > 0, such as https://qms.nextgis.com/geoservices/670/, the map preview may be blank because the service is not available globally. In such cases, it is also difficult to pan the map in order to see a meaningful preview.

An optional initial view definition can be useful for people browsing the QMS catalog. In order to maximize value of the preview, the initial zoom would also be part of the initial view setting.

The default view would remain lat=0, lon=0, zoom=minZoom

**Comment by simgislab** [comment]
yes, we were thinking about automatically changing zoom for a preview.

The trick is where to pan automatically, as those services are not always global. I don't think that 0,0 for a starting point is a good idea.
